Content.Server/Punpun/PunpunSystem.cs

```diff
@@ -0,0 +1,109 @@
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Inventory;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Components;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Punpun;
+
+public sealed class PunpunSystem : EntitySystem
+{
+    [Dependency] private readonly InventorySystem _inventory = default!;
+    [Dependency] private readonly ServerMetaDataSystem _meta = default!;
+
+    private (int, string, string) _punpunData = (0, string.Empty, string.Empty);
+
+    // All the roman numerals we'll need to display
+    private readonly Dictionary<int, string> _numerals = new()
+    {
+        { 0, "I" },
+        { 1, "II" },
+        { 2, "III" },
+        { 3, "IV" },
+        { 4, "V" },
+        { 5, "VI" },
+        { 6, "VII" },
+        { 7, "VIII" },
+        { 8, "IX" },
+        { 9, "X" },
+        { 10, "XI" },
+        { 11, "XII" },
+        { 12, "XIII" },
+        { 13, "XIV" }
+    };
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<PunpunComponent, ComponentStartup>(OnRoundStart);
+        SubscribeLocalEvent<RoundEndTextAppendEvent>(OnRoundEnd);
+    }
+
+    // Checks if the Punpun data has any items to equip, and names the Punpun upon initialization
+    private void OnRoundStart(EntityUid uid, PunpunComponent component, ComponentStartup args)
+    {
+        if (_punpunData.Item1 > 13)
```

Comment:
```suggestion
        if (_punpunData.Item1 > component.MaxLives)
```
And the reason I'm asking about porting in a constructor, is that I actually think this 13 here should be instead a variable, because someone might hypothetically want to set it to some absurdly high number. :)

That and 13 on its is a "Magic Number" that breaks our own maintainer standards. It should either be a constant, or better yet a component variable.